index.html

```diff
@@ -13,14 +13,23 @@
                     <h3>Welcome</h3>
                 </a>
                 <nav class="menu">
-                    <img src="assets/img/menu.svg" alt="menu link">
+                    <img src="assets/img/menu.svg" alt="menu link" id="hamburger">
                 </nav>
                 <nav class="desktop-toolbar">
                     <li class="desktop-toolbar__li"><a href="#work-title">Portfolio</a></li>
                     <li class="desktop-toolbar__li"><a href="#about-myself">About</a></li>
                     <li class="desktop-toolbar__li"><a href="#form">Contact</a></li>
                 </nav>
             </div>
+            <nav class="header-nav-mobile hidden">                
```

Comment:
- [ ] Kindly remove those images from the mobile menu to match the Figma design 

| Your design | Figma design | 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/215796036-b34f2cca-0ff5-4d9a-9eb6-14217db4f570.png) | ![image](https://user-images.githubusercontent.com/24703171/215796165-73171112-dbe6-4148-808d-e3f8f547c7a2.png) |